cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -291,7 +293,8 @@ <h2 class="h3 u-mb15">
                     {{ apr(card.intro_apr_median) }}
                 {% endif %}
                 for
-                {{ '%g' | format(card.median_length_of_introductory_apr) }} months
+                {{ '%g' | format(card.median_length_of_introductory_apr | round) }}
```

Comment:
What's the rationale behind adding `round` here? Looking in the current dataset, I only see one example of a non-integer `median_length_of_introductory_apr` -- 14.97 -- and there are no examples in the pre-release dataset. Any reason not to show the actual float value (perhaps limited to a reasonable number of decimal places) if that's actually what's provided in the data?

If you would prefer to round, a more concise way to do this would be:

```suggestion
                {{ '%.0f' | format(card.median_length_of_introductory_apr) }}
```

Reply:
I found a few cards with non-integer median lengths in the June 2023 dataset, [this one](http://localhost:8000/consumer-tools/credit-cards/explore-cards/cards/bmo-harris-bank-national-association-cash-back-credit-card/) being the prime example with `"median_length_of_introductory_apr": 14.97` and `"median_length_of_balance_transfer_apr": 14.1`. The thinking was seeing such a precise length might confuse consumers used to seeing a nice round "0% for 14 months" on offers, and since there were only a handful of cards with non-integer lengths, we were OK with rounding those.

I did simplify how we're doing the rounding in b359235 like you suggested.